lib/expectations_runner.rb

```diff
@@ -19,9 +25,18 @@ class Mumukit::Inspection::PlainInspection
 
   def expectations
     {
-      'HasWhile' => /AST\(while/
+      'HasWhile' => use(/AST\(while/),
```

Comment:
Unfortunately,  you cannot avoid parentheses on function calls inside hash declarations, but you can move them like `(use /AST\(while/  )`